src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendApplicationAboutToSubmitHandlerTest.java

```diff
@@ -233,5 +313,15 @@ private CallbackRequest doValidCaseDataSetUp(final String path) {
         }
     }
 
+    private FinremCallbackRequest buildCallbackRequest() {
+        return FinremCallbackRequest
+            .builder()
+            .eventType(EventType.AMEND_APP_DETAILS)
+            .caseDetailsBefore(FinremCaseDetails.builder().id(123L).caseType(CONSENTED)
+                .data(new FinremCaseData()).build())
+            .caseDetails(FinremCaseDetails.builder().id(123L).caseType(CONSENTED)
+                .data(new FinremCaseData()).build())
+            .build();
+    }
```

Comment:
Not sure if you can reuse the `buildConsentCallbackRequest` defined in the parent class.

Reply:
I think you can simply reuse the parent method by

`return buildConsentCallbackRequest(EventType.AMEND_APP_DETAILS);`

Reply:
[cb00a1a](https://github.com/hmcts/finrem-case-orchestration-service/pull/1753/commits/cb00a1afefa3c10beac90cecd3181e7230463add)

Reply:
[cb00a1a](https://github.com/hmcts/finrem-case-orchestration-service/pull/1753/commits/cb00a1afefa3c10beac90cecd3181e7230463add)